src/pwm.rs

```diff
@@ -561,12 +561,26 @@ pins! {
         CH1(ComplementaryDisabled): [
             PA2<Alternate<AF9>>,
             PB14<Alternate<AF1>>,
-            PF9<Alternate<AF3>>
+            PF9<Alternate<AF3>>,
+
+            #[cfg(feature = "pwm-open-drain")]
+            PA2<AlternateOD<AF9>>,
```

Comment:
What about changing the pin! macro so that we only need to specify pin and alternate function here? The macro would then generate the things both for `Alternate` and `AlternateOD`. This would avoid some repetition, not sure if it is worth the extra complexity though. Just a though...

So instead of having to specify

```rust
PA2<Alternate<AF9>>,
PA2<AlternateOD<AF9>>,
```
, something like this would suffice
```rust
 PA2: AF9,
```

Reply:
I think that's a good suggestion. I will pick this up after the Christmas holidays. Thank you.

Reply:
I changed the pin macro as you suggested.

Reply:
Great! :)